tests/experimental/pydantic/test_basic.py

```diff
@@ -172,6 +172,40 @@ class UserType:
     assert definition.fields[2].is_optional is False
 
 
+@pytest.mark.xfail(
+    reason=(
+        "passing default values when extending types from pydantic "
+        "is not currently supported"
+    )
+)
+def test_type_with_fields_coming_from_strawberry_and_pydantic_with_default():
```

Comment:
basically a reminder for adding support for this in a next release

Reply:
Can you make the reason link to/reference a GitHub issue#

Reply:
Will do in the morning before merging it 😊